frontend/src/queries/nodes/DataTable/dataTableLogic.ts

```diff
@@ -84,16 +114,26 @@ export const dataTableLogic = kea<dataTableLogicType>([
                                         [categoryRowKey]: dayjs(result[orderKeyIndex]).format('LL'),
                                     })
                                 }
-                                newResults.push(result)
+                                newResults.push(convertResultToDisplayedColumns(result))
                                 lastResult = result
                             }
                             return newResults
+                        } else {
+                            return results.map((result) => convertResultToDisplayedColumns(result))
                         }
                     }
                 }
                 return response && 'results' in response ? (response as any).results ?? null : null
             },
         ],
+        isRowHighlighted: [
```

Comment:
Hm I can't quite grasp the purpose of this highlighting from the code. Can you add a one-line comment explaining what rows we want highlighted?

Reply:
The row data structure refactor removed the need for this proxying. `yeet!`